src/wallet/wallet.cpp

```diff
@@ -1862,6 +1862,8 @@ CWallet::ScanResult CWallet::ScanForWalletTransactions(const uint256& start_bloc
         } else if (block_height && chain().shutdownRequested()) {
             WalletLogPrintf("Rescan interrupted by shutdown request at block %d. Progress=%f\n", *block_height, progress_current);
             result.status = ScanResult::USER_ABORT;
+        } else {
+            WalletLogPrintf("Rescan completed.\n");
         }
     }
```

Comment:
style-nit: Could add a commit to remove this scope (and the 4 extraneous spaces that precede the lines)?

Reply:
I added the commit to do that, but it the diff is pretty big.